packages/unplugin/src/plugin/core.ts

```diff
@@ -115,15 +115,7 @@ export const unpluginFactory: UnpluginFactory<PandaPluginOptions | undefined> =
     if (outfile !== ids.css.resolved) {
       getCtx().then((ctx) => fs.writeFile(outfile, ctx.toCss(ctx.panda.createSheet(), options)))
     }
-
-    const timestamp = Date.now()
-    server.moduleGraph.invalidateModule(mod, new Set(), timestamp, true)
```

Comment:
There's a similar line on 247 responding to changes in Vite. It seems like the invalidation here should be enough though, since this code will also get called when files change.

Reply:
Yeah that's for a different code path of module invalidation and didn't want to change too much in scope

Reply:
let's release it with this for now; we can always revisit that part later on if needed
I'll try on my end as well in a realworld project 

btw we might want to add https://github.com/stackblitz-labs/pkg.pr.new if you guys want to contribute regularly

Reply:
Oh wow. Yeah that would be awesome, didn't know that was a thing.